admin/sql/CreateIndexes.sql

```diff
@@ -456,6 +456,9 @@ CREATE UNIQUE INDEX place_attribute_type_allowed_value_idx_gid ON place_attribut
 
 CREATE UNIQUE INDEX place_alias_type_idx_gid ON place_alias_type (gid);
 
+CREATE INDEX place_rating_raw_idx_place ON place_rating_raw (place);
```

Comment:
This index is actually redundant with `place_rating_raw_pkey`, so it can be dropped. (PostgreSQL [automatically creates a unique index for primary keys](https://www.postgresql.org/docs/9.4/indexes-unique.html), and the primary key here already has `place` as the left-most column.)

The other `${entity}_rating_raw_idx_${entity}` indexes can be dropped too, but it's not something urgent we have to do this schema change.

Reply:
But would it be ok to do it this schema change? Or is it something we should have announced beforehand?

Reply:
Also: "The index covers the columns that make up the primary key or unique constraint (a multicolumn index, if appropriate), and is the mechanism that enforces the constraint." - shouldn't that mean that `place_rating_raw_idx_editor` is also covered? Or does it matter that it is the first vs second column, and why?

Reply:
> But would it be ok to do it this schema change?

It would be okay IMO, since they're redundant.

> shouldn't that mean that place_rating_raw_idx_editor is also covered? Or does it matter that it is the first vs second column, and why?

You can use the pkey index to query for editors, but it's less efficient because it's not the leading column, so more of the table has to be scanned. For B-tree indexes (which is the default), the "why" is just the fact of that being how the tree is sorted. https://www.postgresql.org/docs/12/indexes-multicolumn.html

Reply:
(I specified B-tree here 'cause it does depend on the index type; as that link notes, with GIN/BRIN indexes the column order doesn't affect search effectiveness.)

Reply:
Added https://github.com/metabrainz/musicbrainz-server/pull/2091 for that